dart/constraint/ContactConstraint.cpp

```diff
@@ -950,5 +1027,34 @@ void ContactConstraint::uniteSkeletons()
   }
 }
 
+//==============================================================================
+double ContactConstraint::getPrimarySlipCompliance() const
+{
+  return mSlipCompliance;
+}
+
+//==============================================================================
+void ContactConstraint::setPrimarySlipCompliance(double slip)
+{
+  mSlipCompliance = slip;
+}
+
+//==============================================================================
+double ContactConstraint::getSecondarySlipCompliance() const
+{
+  return mSecondarySlipCompliance;
+}
+
+//==============================================================================
+void ContactConstraint::setSecondarySlipCompliance(double slip)
+{
+  mSecondarySlipCompliance = slip;
+}
+
+//==============================================================================
+const collision::Contact& ContactConstraint::getContact() const
+{
+  return mContact;
+}
```

Comment:
Nit: Please add an empty line after this line

Reply:
https://github.com/dartsim/dart/commit/5d81fbbc3a61d493f7f4f201657aa482ed169d2c